website/static/overrides.css

```diff
@@ -523,7 +523,7 @@ a {
 }
 
 .hljs {
-  padding: 16px;
```

Comment:
Let's not change these styles. Instead please move the button up (`top: -5px; right: 16px` so that it doesn't overlap) and show it only when the code block is hovered.